src/data/community-events.json

```diff
@@ -89,6 +89,15 @@
     "startDate": "2023-05-23",
     "endDate": "2023-05-24"
   },
+  {
+    "title": "SpaghettETH II",
+    "to": "https://naples.spaghett-eth.com/",
+    "sponsor": "EF, Ambire, Zerion, ETHLocal, Yomi",
```

Comment:
Is the sponsors list checked? I see [some differences](https://drive.google.com/file/d/1uYCTbIK4AxjfmCDD34pWC4L7uQzM1SM_/view) in the pdf

Reply:
those on the pitchdeck are the sponsors of the first edition, the second edition ones are listed on the website.

Reply:
Ok, will approve then. FYI the sponsor button from the hero leads to the PDF so could be a bit confusing.

Reply:
yes because "sponsor" is meant as a verb, therefore it leads to the deck, while the "sponsors" section lists the actual sponsors

Reply:
got it